packages/realm/src/Configuration.ts

```diff
@@ -82,10 +85,10 @@ type BaseConfiguration = {
 //   Manual = "manual",
 // }
 
```

Comment:
having this piece of code caused name collision, might need to double-check that it's not used elsewhere..

Reply:
The bindgen project is "promoting" TypeScript to be first-class citizen, and it might have some implications of how the TypeScript API is described. Thank you for raising it - and we should double-check if we can remove this code section.